### ISSUE TITLE
[Accessibility] [Supporting the platform -TSDocs]: Focus indicator is not moving to the displayed search results in high-contrast mode. [Sev3]

### ISSUE THREAD

**Issue by Laxmidurga** [microsoft/tsdoc.org-website#42]
**Test Environment:**
OS Build: Windows 11
Version: 21H2 (OS Build 22000.318)
Browser: Edge
Version 96.0.1054.43 (Official build) (64-bit)
URL: https://tsdoc.org/
Screen reader: Narrator

**Pre-requisites:**
Settings > Accessibility > Contrast themes > Select ‘Aquatic’ and 'Dessert' modes.

**Repro Steps:**
1.	Open the above URL and login with valid credentials.
2.	TSDoc page will get displayed.
3.	Now navigate to search docs field type any letter in the field (ex, micr), suggestions will appear.
4.	Now, enable high-contrast mode in settings and check whether the focus indicator is visible on the displayed suggestions or not.

**Actual:**
Focus indicator is not moving to the displayed search results in high-contrast mode.

**Expected:**
Focus indicator should move to the displayed search results in high-contrast mode.

**User Impact:**
When focus indicator is not visible on the interactive elements in high-contrast mode, customers who rely on that feature are blocked from information and functionality of the full user experience within the site/app.

**Guideline Reference:**
EN 301 549 V3.2.1 - 11.6.2.0 No disruption of accessibility features

https://user-images.githubusercontent.com/88436727/151990850-0118d43d-17df-4b2f-a020-cb87f75ebf54.mp4

**Comment by octogonz** [comment]
This issue has been closed because this Jekyll project has been retired. The documentation content has been migrated to Docusaurus. 

If the problem has not been addressed, please file a new issue under this GitHub project instead:  

https://github.com/microsoft/rushstack-websites
